packages/design-system-docs/src/pages/components/UsaBanner/usabanner.example.html

```diff
@@ -0,0 +1,43 @@
+<div class="ds-c-usa-banner">
+  <header class="ds-c-usa-banner__header">
+    <img class="ds-c-usa-banner__header-flag" src="/images/us_flag_small.png" alt="U.S. flag" />
```

Comment:
This is using the png in the doc site instead of the svg from the core. I think all the images in the HTML example should be updated to use the core svg files